### PR TITLE
DISPATCH-1799: test message body_data functions

### PR DESCRIPTION
**Pull request by ChugR**
First pass at setting various vbin chunk sizes, vbin number-of-chunks. This is then cast across the differing buffer sizes provided by the unit_test_size_XXX test group.

### CODE REVIEW

**Review by kgiusti — APPROVED** [review]

File: tests/message_test.c [hunk 1]
```diff
@@ -703,6 +705,239 @@ static char *test_check_weird_messages(void *context)
...
+        // Create 'buf2' as a linear buffer of the raw data to be sent.
+        // Buffer filled with chunk index + 1.
+        unsigned char *buf2 = (unsigned char *)malloc(chunk_size);
+        memset(buf2, j+1, chunk_size);
+
+        // Use 'set_content' to convert raw buffer 'buf2'
+        // into a buffer list in message 'mule'.
+        qd_message_t *mule = qd_message();
+        qd_message_content_t *mule_content = MSG_CONTENT(mule);
+        set_content(mule_content, buf2, chunk_size);
```
Comment (ganeshmurthy) [inline tests/message_test.c:733]: Where is this set_content() function ?

Comment (ganeshmurthy) [reply tests/message_test.c:733]: Oops, found it.

Comment (ganeshmurthy) [reply tests/message_test.c:733]: > Where is this set_content() function ?

Oops, found it, sorry

**Review by ganeshmurthy — APPROVED** [review]

**Comment by ChugR** [comment]
Fixed at commit aef58e8

**Pull request closed without merging**